contracts/rate_oracles/BaseRateOracle.sol

```diff
@@ -216,7 +202,7 @@ abstract contract BaseRateOracle is IRateOracle, Ownable {
 
         if (rateToRay > rateFromRay) {
             uint256 result = WadRayMath.rayToWad(
-                WadRayMath.rayDiv(rateToRay, rateFromRay) - WadRayMath.RAY
+                getGrowthBetween(rateFromRay, rateToRay)
```

Comment:
for readability maybe worth naming this function getRateOfReturn(rateFromRay, rateToRay)

Reply:
changed [c2a5c2f](https://github.com/Voltz-Protocol/voltz-core/pull/230/commits/c2a5c2fda8ffd7f8ede64ea6b3b1d51a822d32d0)